### PR TITLE
Update API reference for 3.12

### PR DESCRIPTION
**Pull request by vicferpoy**
## Description

`User manual > API > reference.rst` has been updated for v3.12. This `.rst` has been autogenerated with our script and manually reviewed.

## Checks
- [x] It compiles without warnings.
- [x] Spelling and grammar. 

Regards.

### CODE REVIEW

**Comment by alberpilot** [comment]
In addition to the previous comment, please update your branch in order to solve conflicts. Thanks

**Comment by vicferpoy** [comment]
Hello @davidfmiranda . 
The `.. code-block:: json :class: output` block was added recently and our script was not prepared for it. I have modified the script and it should implement this block from now on.

Regards.

File: source/user-manual/api/reference.rst [hunk 1]
```diff
@@ -6603,19 +6721,13 @@ Returns the CIS requirements of all rootchecks of the specified agent.
...
-   {
-      "error": 0,
-      "data": {
-         "totalItems": 2,
-         "items": [
-            "2.3 RHEL7",
-            "1.4 RHEL7"
-         ]
-      }
-   }
```
Comment (crd1985) [inline source/user-manual/api/reference.rst]: Preserve previous example

File: source/user-manual/api/reference.rst [hunk 2]
```diff
@@ -6707,19 +6819,13 @@ Returns the PCI requirements of all rootchecks of the agent.
...
-   {
-      "error": 0,
-      "data": {
-         "totalItems": 2,
-         "items": [
-            "4.1",
-            "2.2.4"
-         ]
-      }
-   }
```
Comment (crd1985) [inline source/user-manual/api/reference.rst]: Preserve previous example

**Pull request merged**